module/VuFind/src/VuFind/View/Helper/Root/Component.php

```diff
@@ -52,6 +52,8 @@ class Component extends AbstractHelper
      */
     public function __invoke(string $name, $params = []): string
     {
+        static $invokation = 0;
```

Comment:
Nitpick: should probably be spelled "invocation".

Reply:
Fixed!